tests/unit/nupic/algorithms/anomaly_test.py

```diff
@@ -115,5 +116,21 @@ def testComputeAnomalySelectModePure(self):
 
 
 
+  def testSerialization(self):
+    """serialization using pickle"""
+    # instances to test
+    aDef = Anomaly()
+    aLike = anomaly.Anomaly(mode=Anomaly.MODE_LIKELIHOOD)
+    aWeig = Anomaly(mode=Anomaly.MODE_WEIGHTED)
+#    aCust = Anomaly(mode=Anomaly.MODE_CUSTOM, customComputeFn=sum)
+    inst = [aDef, aLike, aWeig] #, aCust] #TODO add soon
+
+    for a in inst:
```

Comment:
Also need to validate that `self._computeAnomaly` is restored properly and is actually the expected method; in case of non-custom, need to check that it matches the identity of the expected instance method; e.g.,

```
if restored._mode == restored.MODE_PURE:
    self.assertIs(restored._computeAnomaly, restored. _computeRaw)
elif restored._mode == ...:
    ...
```